homeassistant/components/climate/opentherm_gw.py

```diff
@@ -1,34 +1,23 @@
 """
-Support for OpenTherm Gateway devices.
+Support for OpenTherm Gateway climate devices.
 
-For more details about this component, please refer to the documentation at
-http://home-assistant.io/components/climate.opentherm_gw/
+For more details about this platform, please refer to the documentation at
+http://home-assistant.io/components/opentherm_gw/
```

Comment:
The url is wrong. Revert to old url. Each platform should have its own docs page.